<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setPwdInfor</class>
 <widget class="QWidget" name="setPwdInfor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#shadowbg{
    background-color:#FFFFFF;
    border-radius: 4px;
	    font-family:微软雅黑;
	border:1px solid #DADADA80
}

QLabel#label_title,QLabel#label_desc,QLabel#label_pwd{
    font-size:12px;
	font-family:微软雅黑;
    color:#000000;
}
QPushButton{
	color:#333333;
	font:bold 12px;
}
QPushButton#Btn_Cancel{
   color: #666666;
	background-color: #FFFFFF;
	border:none;
}

QPushButton#Btn_Cancel:hover{
    border:1px solid #EBEBEB;
    background-color:#EBEBEB;
}


QPushButton#Btn_Cancel:pressed{
    border:1px solid #D6D6D6;
    background-color:#D6D6D6;
}


QPushButton#Btn_OK{
    color: #666666;
	background-color: #FFFFFF;
	border:none;
}
QPushButton#Btn_OK:hover {
    color: #FFFFFF;
	background-color: #3dbbca;
}
QPushButton#Btn_OK:pressed{

    color: #FFFFFF;
	background-color: #1194a4;
}

QTextEdit:focus{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}

QLineEdit#lineEdit_title,#lineEdit_Pwd,#lineEdit_desc,#lineEdit_Pwd_second
{
	background:#FFFFFF;
	color:#333333;
	font:bold 12px;
}
QLineEdit#lineEdit_title:hover,#lineEdit_Pwd:hover,#lineEdit_desc:hover,#lineEdit_Pwd_second:hover
{
    border:1px solid #999999;
}
QLineEdit#lineEdit_title:focus,#lineEdit_Pwd:focus,#lineEdit_desc:focus,#lineEdit_Pwd_second:focus
{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}

QLabel#label
{
	border:none;
	color:#FF0000;
	font-size:13px;
	font-weight:bold;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="TitlebarWidget" name="titlebar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_infor" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_titlewidget" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>80</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>80</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>80</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_title">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="placeholderText">
                     <string>密码箱名称</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_desc" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>40</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>80</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>80</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_desc">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="inputMethodHints">
                     <set>Qt::ImhNone</set>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>密码箱的描叙</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_pwd" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>80</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>80</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>80</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_Pwd">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>输入主密码</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_pwd_second" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>40</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>80</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>80</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_Pwd_second">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>验证主密码</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>85</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>85</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>40</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>80</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::RichText</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>50</number>
               </property>
               <property name="leftMargin">
                <number>200</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>60</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="Btn_Cancel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>取消</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Btn_OK">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>确定</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
